modules/ROOT/pages/getting-started.adoc

```diff
@@ -92,21 +89,21 @@ how to do this.
 == Package layering
 
 Package layering works by modifying your Silverblue installation. As the name 
-implies, it works by modifying the packages from which Silverblue is composed. 
-Any software package that is provided by Fedora can be installed on Silverblue 
-using this method.
+implies, it works by extending the packages from which Silverblue is composed. 
+
+Good examples of packages to be layered would be:
 
-A Silverblue installation with packages layered on top loses some of the 
-advantages of immutability: with layering in place, the system is no longer 
-identical to other installations, and is not as robust.
+ - `fish`: An alternative Unix shell
+ - `sway`: A Wayland tiling compositor
+ - `libvirt`: The libvirt daemon
 
-Nevertheless, package layering can be a useful option for installing 
-software that might not be available through other means, or for installing 
-software that, for technical reasons, needs to be part of the host operating 
-system.
+Most (but not all) RPM packages provided by Fedora can be installed on Silverblue 
+using this method.
 
-Typically, the system must be rebooted after a package has been layered. This 
-is an practical disadvantage compared with installing Flatpaks or containers.
+Currently, using package layering creates a new "deployment", or bootable filesystem
+root.  It does not affect your current root.  This preserves rollback and the transactional model, but means system must be rebooted after a package has been layered.
+Eventually this limitation may be lifted, but it's generally expected that you use package
```

Comment:
Not sure we need to talk about vague future ideas here. Lets stick to documenting layering as it exists today.

Reply:
Similar text already existed before; it feels worth at least mentioning this as it's a common topic, and here we're also explaining why the development focus is on apps/containers rather than live changes on the host.